docs/how-to/using_rocm_bandwidth_test.rst

```diff
@@ -27,156 +29,152 @@ ROCm Repo Forest
 
-The above command will print Version of RBT and time taken to perform copy for given device list.
+The preceding command prints the RBT version and time taken to perform copy for the given device list.
 
 Data path validation test
-==============================
+##############################
 
-Run this test to validate data path from a source device to destination device by by copying data,
+To validate data path from a Src to Dst device by copying data, use:
 
 .. code-block::
 
       <shell_prompt> ./rocm_bandwidth_test -v
 
-The above command will print the following four details: 
+The preceding command prints the following details:
 
-* Version of ROCm Bandwidth Test (RBT)
+* RBT version
 * List of devices
-* Access matrix 
-* Data Path Validation among the various devices
-
+* Access matrix
+* Data path validation among the various devices
 
-Default unidirectional & bidirectional all devices bandwidth test
-==================================================================
+Default unidirectional and bidirectional bandwidth test for all devices
+##########################################################################
 
-Run the following test to collect performance characteristics of unidirectional and bidirectional copy operations involving ALL devices of a given Rocm platform.
+To collect performance characteristics of unidirectional and bidirectional copy operations involving `all` devices on a given Rocm platform, use:
 
 .. code-block::
 
       <shell_prompt> ./rocm_bandwidth_test
 
-The above command will issue unidirectional and bidirectional copy operations among all the devices of the platform.
+The preceding command issues unidirectional and bidirectional copy operations among all the devices of the platform.
 
-Host-to-Device (H2D) bandwidth
-================================
+Host-to-Device bandwidth
+##################################
 
-Run the test below to collect performance characteristics of H2D copy operations of a given ROCm platform.
+To collect performance characteristics of Host-to-Device (H2D) copy operations on a given ROCm platform, use:
 
 .. code-block::
-            
+
             <shell_prompt> ./rocm_bandwidth_test -s <cpu_dev_IdX>,<cpu_dev_IdY>,- - - -d <gpu_dev_IdM>,
             <gpu_dev_IdN>, - - -
 
-The above command will issue unidirectional copy operations between Src and Dst devices. Specifically it will pair each device of Src List it
-with each device of Dst List i.e. it will launch sizeof(SrcList) x sizeof(DstList) number of copy operations. It is assumed that user has
+The preceding command issues unidirectional copy operations between Src and Dst devices. To be specific, it pairs each device from the Src list
+with each device from the Dst list. This implies that the command launches sizeof(SrcList) x sizeof(DstList) number of copy operations. It is assumed that user has
 determined access from Src device to Dst device exists by consulting device access matrix.
```

Comment:
I can't understand what this means: "It is assumed that user has
determined access from Src device to Dst device exists by consulting device access matrix." Could you please rephrase this.

Reply:
This means that `source` and `target` need to be valid devices (device numbers), which can be checked by looking at the device matrix (or topology) output.

Reply:
Added "Ensure to validate the Src and Dst devices, their device numbers, and the connection between them by looking at the device matrix (or topology) output."